src/dotnet/Chat.UI.Blazor/Services/ChatListUI.cs

```diff
@@ -11,6 +11,7 @@ public partial class ChatListUI : WorkerBase, IHasServices, IComputeService, INo
     public static readonly int AllItemCountWhenLoading = 14;
     private static readonly TimeSpan MinNotificationInterval = TimeSpan.FromSeconds(5);
 
+    // modified implicitly after returned by GetItems
```

Comment:
Comments are like logged messages - sentence-like, but ok to omit the ending ".", coz they are typically a part of the "list".